UserManual.md

```diff
@@ -660,6 +669,41 @@ Laminar will serve a job's current status as a pretty badge at the url `/badge/J
 
 # Reference
 
+## LAMINAR_HOME folder scheme
+
+```
+$LAMINAR_HOME/
+├── cfg/
+│   ├── before
+│   ├── jobs/
+│   │   ├── $JOB.init
+│   │   ├── $JOB.env
+│   │   ├── $JOB.before
+│   │   ├── $JOB.run
+│   │   ├── $JOB.after
+│   │   └── $JOB.conf
+│   ├── after
+│   │   scripts/
+│   │   ├── foo
+│   │   └── bar
+│   ├── env
+│   ├── contexts/
+│   │   ├── $CONTEXT.env
+│   │   └── $CONTEXT.conf
+│   └── groups.conf
+├── archive/
+│   └── $JOB/
+│       └── $RUN/           # $ARCHIVE
+├── run/
+│   └── $JOB/
+│       └── workspace/      # $WORKSPACE
+├── custom/
+│   └── index.html
+└── laminar.sqlite
+```
+
+`LAMINAR_HOME` and its content need to be at least owned by the user specified in `laminar.service`, which defaults to the system user `laminar`.
```

Comment:
This is not necessarily true, admins may want a more nuanced set of permissions. For example, the content of `cfg` and `custom` only needs to be readable/executable by the user running `laminard`, not owned or writable.

Perhaps something like:

The user running `laminard` (by default the system user `laminar`) must have read/write access to `laminar.sqlite`, read/execute access to the contents of `cfg`, and write access to `archive` and `run` if the running jobs are to have the ability to archive artifacts or utilize the workspace respectively.

Or is that too complicated.... :shrug:

Reply:
I had hoped for that level of detail, thank you!

No, I think this perfectly understandable.

I wonder about having multiple `LAMINAR_HOME`s, though. :thinking:
E.g. for a server running multiple services not linked to each other.

Would you do a
```bash
LAMINAR_HOME=/path/to/service-laminar-home/ laminarc ...
```
for every service?
And maybe symlinking (with which permissions)?

Reply:
Having multiple instances of `laminard` (and `LAMINAR_HOME`) is totally OK, but setting it up is a bit out of scope since there certainly isn't a one-size-fits-all approach.

 > laminar.conf needs read permission for the laminar.service user, right?

No. systemd just expands everything in `laminar.conf` as environment variables to `laminard`, you can just as well run `$ LAMINAR_HOME=/foo/bar LAMINAR_BIND_HTTP=*:1234 /path/to/laminard`, or equivalently in whatever init system or orchestration tool you like